assets/bundles/bundle_zh_CN.properties

```diff
@@ -1,147 +1,343 @@
+unit.cyber-io-holo-fighter.description= 全息投影器构建的短命打手。
+unit.cyber-io-holo-guardian.name=全息防御机
+unit.cyber-io-holo-guardian.description=全息投影器构建的用于保护Plum Star Inc.的保安。尽职尽责，誓死不悔。
+unit.cyber-io-holo-architect.name=全息建造机
+unit.cyber-io-holo-architect.description=全息投影器构建的建造机，但不是被建造出来的哦。
+unit.cyber-io-holo-supporter.name=全息辅助机
+unit.cyber-io-holo-supporter.description=全息投影器构建的辅助机，但却不能辅助自己。
 # Bars
-bar.cyber-io.generation=\u751F\u6210 {0}
-bar.cyber-io.slow-down=\u901F\u5EA6\u500D\u7387: -{0}%
-bar.cyber-io.efficiency-absorption=\u6548\u7387\u5438\u6536: {0}%
-bar.cyber-io.range=\u8303\u56F4\uFF1A{0} \u4E2A\u65B9\u5757
-bar.cyber-io.cool-down=\u51B7\u5374\uFF1A {0}s
-bar.cyber-io.is-alive=\u5B58\u6D3B\uFF1A {0}
-bar.cyber-io.is-projecting=\u6295\u5F71\uFF1A{0}
-bar.cyber-io.rest-restore=\u5269\u4F59\u590D\u539F\uFF1A{0}
-bar.cyber-io.charge=\u5145\u80FD\uFF1A {0}s
-bar.cyber-io.last-damaged=\u4E0A\u6B21\u4F24\u5BB3\uFF1A {0}s
-bar.cyber-io.prism=\u68F1\u955C
-bar.cyber-io.linked=\u5DF2\u8FDE\u63A5
-bar.cyber-io.unlinked=\u672A\u8FDE\u63A5
-bar.cyber-io.receiver=\u63A5\u6536\u7AEF
-bar.cyber-io.sender=\u53D1\u9001\u7AEF
-bar.cyber-io.ani-state-last=\u6301\u7EED\u65F6\u95F4 {0}
+bar.cyber-io.generation=生成：{0}
+bar.cyber-io.slow-down=减速：-{0}%
+bar.cyber-io.efficiency-absorption=效率吸收： {0}%
+bar.cyber-io.range=范围：{0}个方块
+bar.cyber-io.cool-down=冷却：{0}秒
+bar.cyber-io.is-alive=是否存活：{0}
+bar.cyber-io.rest-restore=剩余修复：{0}
+bar.cyber-io.prism=棱镜
+bar.cyber-io.linked=已连接
+bar.cyber-io.unlinked=未连接
+bar.cyber-io.receiver=接收端……
+bar.cyber-io.sender=发送端
 bar.cyber-io.null=Null
-bar.cyber-io.client=\u5BA2\u6237\u7AEF
-bar.cyber-io.host=\u4E3B\u673A
-bar.cyber-io.rest-life=\u5269\u4F59\u5BFF\u547D
-bar.cyber-io.rest-life.figure=\u5269\u4F59\u5BFF\u547D\uFF1A {0}s
-bar.cyber-io.rest-resurrect=\u5269\u4F59\u590D\u6D3B\u6B21\u6570\uFF1A {0} \u6B21
+bar.cyber-io.client=获取端
+bar.cyber-io.host=提供端
+bar.cyber-io.rest-life=剩余寿命
+bar.cyber-io.rest-life.figure=剩余寿命：{0}秒
+bar.cyber-io.rest-resurrect=剩余复活次数：{0} 次
+bar.cyber-io.temperature=温度
+# Statistic and Category
+category.cyber-io-heimdall=海姆达尔
+category.cyber-io-data-transfer=数据传输
+stat.cyber-io-heimdall-basic=基本
+stat.cyber-io-heimdall-improve=升级
+stat.cyber-io-data-transfer-range=范围
+stat.cyber-io-data-transfer-max-sender=最多发送端
+stat.cyber-io-data-transfer-max-receiver=最多接收端
+stat.cyber-io-data-transfer-max-host=最多提供端
+stat.cyber-io-data-transfer-max-client=最多获取端
+stat.power-cyber-io-transfer-speed=传输速度
 # Bundles
-cyber-io.cost.second=\u79D2
-cyber-io.cost.minute=\u5206
-cyber-io.speed-increase.range=\u4ECE {0}% \u5230 {1}%
+cyber-io.cost.second=秒
+cyber-io.cost.minute=分
+cyber-io.speed-increase.range=从 {0}% 到 {1}%
+cyber-io.full-sender=已达到发送端最大数量
+cyber-io.full-receiver=已达到接收端最大数量
+cyber-io.full-host=已达到提供端最大数量
+cyber-io.full-client=已达到获取端最大数量
+cyber-io.over-range=超过范围
+cyber-io.unsaved-change=你还没有保存更改
+cyber-io.unlimited=无限
+cyber-io.random-name=ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxy%&#~*|?!
```

Comment:
This key is used to generate a random name for TMTRAINER. I'd like to see the localized one.
It will be split into characters.